code/game/jobs/access.dm

```diff
@@ -601,3 +601,13 @@ proc/get_all_job_icons() //For all existing HUD icons
 
 	return "Unknown" //Return unknown if none of the above apply
 
+/obj/proc/HasPermit() //Used in secHUD icon generation
+	var/obj/item/weapon/card/id/I = src
+
+	if(I)
+		if(99 in I.access)

```

Comment:
access_weapons is the [constant](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/jobs/access.dm#L78) for weapons access. You're never supposed to use the magic numbers because it's shit to read.